lib/src/range.dart

```diff
@@ -0,0 +1,8 @@
+part of many_points;
+
+class Range {
```

Comment:
Lets make this an immutable class.

class Range {
  final num min;
  final num max;

  const Range(this.min, this,max);
}